File: algorithm/python/BOJ2577.py
# from https://www.acmicpc.net/problem/2577

A = int(input())
B = int(input())
C = int(input())

digit_count = [0] * 10
result = str(A * B * C)
for i in range(0, len(result)):
    digit_count[int(result[i])] += 1

for i in range(0, 10):
    print(digit_count[i])

File: algorithm/typescript/BOJ2577.ts
// from https://www.acmicpc.net/problem/2577

import fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().split("\n");

const A = Number(input[0]);
const B = Number(input[1]);
const C = Number(input[2]);
const result = String(A * B * C);
const digit = Array(10).fill(0);

for (let i = 0; i < result.length; i++) {
	const idx = Number(result.charAt(i));
	digit[idx]++;
}

console.log(digit.join("\n"));